docs/Generating and adding ssh key.md

```diff
@@ -0,0 +1,76 @@
+## About the file
```

Comment:
This can be a section in the contributors guide if it this is common .
Have this a section linked to instruction on setting up kobman development environment